.github/workflows/gradle-run-tests.yml

```diff
@@ -20,13 +20,17 @@ jobs:
           cache: gradle
       - name: Validate Gradle wrapper
         uses: gradle/wrapper-validation-action@e6e38bacfdf1a337459f332974bb2327a31aaf4b
+      - name: Build Native dependencies
+        run: make
       - name: Test with Gradle
-        run: ./gradlew test
+        run: ./gradlew allTests
       - name: Android Instrumentation Tests
         uses: reactivecircus/android-emulator-runner@v2
         with:
           api-level: 29
-          script: ./gradlew cryptography:connectedAndroidTest
+          script: |
+            ./gradlew cryptography:connectedAndroidTest
+            ./gradlew persistence:connectedAndroidTest
```

Comment:
Just a suggestion
```suggestion
          run: |
            ./gradlew cryptography:connectedAndroidTest
            ./gradlew persistence:connectedAndroidTest
```

Reply:
No can do.

`script` is an argument taken by the [`android-emulator-runner`](https://github.com/ReactiveCircus/android-emulator-runner) action.

It is not a regular `run` command from GitHub Actions.